### PR TITLE
remove dead code

### PR DESCRIPTION
**Pull request by zanuka**
- project cleanup
- added comments
- removed dead code

### CODE REVIEW

**Comment by matt-auckland** [comment]
@zanuka Could you start adding @ro-savage and I as reviewers on these PRs so we can begin getting visibility of the codebase

File: app/middleware/adapter.go [hunk 1]
```diff
@@ -4,8 +4,16 @@ import (
...
 type Adapter func(http.Handler) http.Handler
 
+// existing http.Handler is passed in and the Adapter will adapt it
+// and return a new (wrapped) http.Handler to use in its place
+// To make the adapters run in the order in which they are specified
+// you could reverse through them in the Adapt function
+
+// in essence, this middleware allows us to run code before
+// and/or after our handler code in a HTTP request lifecycle
 func Adapt(h http.Handler, adapters ...Adapter) http.Handler {
```
Comment (matt-auckland) [inline app/middleware/adapter.go]: Just thought I would mention that this is gibberish to me 😛 (just to give you an idea of my level of knowledge around api writing in typed languages.

Comment (zanuka) [reply app/middleware/adapter.go]: in time, it will all make sense 🦅

**Comment by zanuka** [comment]
cheers guys

**Pull request merged**